src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SNI/SNIMarsHandle.cs

```diff
@@ -222,7 +236,7 @@ private uint InternalSendAsync(SNIPacket packet, SNIAsyncCallback callback)
         /// <returns>SNI error code</returns>
         private uint SendPendingPackets()
         {
-            long scopeID = SqlClientEventSource.Log.TrySNIScopeEnterEvent("<sc.SNI.SNIMarsHandle.SendPendingPackets |SNI|INFO|SCOPE>");
+            long scopeID = SqlClientEventSource.Log.TrySNIScopeEnterEvent("SNIMarsHandle.SendPendingPackets | SNI | INFO | SCOPE | Entering Scope {0}");
```

Comment:
Is the scope id implicitly parameter 0 inside the call or does it need passing as an arg?

Reply:
Yes, we need that logged in order to box scopes.